server/package.json

```diff
@@ -11,13 +11,15 @@
         "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
         "prestart": "npx prisma migrate deploy",
         "start": "nest start",
+        "start:testing": "PORT=3001 TESTING_ENV=true nest start",
         "dev": "nest start --watch",
+        "dev:testing": "PORT=3001 TESTING_ENV=true npm run dev",
         "start:debug": "nest start --debug --watch",
         "start:prod": "node dist/main",
         "lint": "eslint \"{src,apps,libs,test}/**/*.ts\"",
         "lint:fix": "npm run lint -- --fix",
         "unit": "jest --watch",
-        "unit:ci": "jest",
+        "unit:ci": "DATABASE_URL=$(bash ./set-testing-db-url.sh) jest",
```

Comment:
Not sure why the unit tests suddenly need a database url, because it worked without before...
Can we get that fixed?

Reply:
Lets just leave it there until it breaks something, then we can get back at it again.